### DISCUSSION TITLE
Airflow 1.10.7 scheduler frequency less than daily not working

### DISCUSSION THREAD

**Discussion by mike-soukup-vividseats** [apache/airflow#36542] [Q&A]
Hello,

I am using Airflow version 1.10.7. I am running into an issue with my scheduler working properly. 

For various DAGs, I set the schedule within the DAG using the `schedule_interval` parameter and cron notation.

However, for any DAG that has a frequency less than daily (i.e. weekly, monthly, etc.) the DAGs are not being scheduled correctly and can only be triggered manually. 

For instance, 

```python
dag = DAG(
    dag_id="dag_name",
    schedule_interval="0 10 1 * *",
    catchup=False,
    default_args=default_args,
    start_date=datetime(2023,5,1),
)
```
^ This DAG is not running monthly as intended. 

```python
dag = DAG(
    'dag_name',
    start_date=days_ago(1),
    schedule_interval='0 13 * * *',
    default_args=default_args,
    catchup=False,
    max_active_runs=1,
)
```
^ This DAG has no problem being scheduled daily.

In short, daily and hourly DAG schedules work just fine. Meanwhile weekly and monthly DAG schedules do not. Can anyone shed light as to why that is the case?

Does this having something to do with the schedule_interval being greater than 1 day / 1 hour for these lower frequency runs?
For instance, if the schedule_interval is set to 0 1 1 * *, would this set the schedule interval to 1 month which would require this DAG to run in the subsequent month? But I still don't think this should be an issue if the start date is set to some previous static date...

Is there a way to validate the schedule timetable for a dag_id to see all future run dates?

Any help would be very much appreciated!

Thank you!

**Comment by raphaelauv** [comment]
```
 I am running into an issue with my scheduler working properly
```
:grin: 

btw probably nobody is going to help you on so old version

**Reply by potiuk** [reply]
Maybe this guy would:

![image](https://github.com/apache/airflow/assets/595491/63c89b51-7233-4101-bc58-884a1bd5572e)

**Reply by potiuk** [reply]
BTW. @mike-soukup-vividseats - Airflow 1.10.15(!) is end-of-life as of 2.5 years now. https://github.com/apache/airflow?tab=readme-ov-file#version-life-cycle

So you would do yourself big favour by upgrading.

**Comment by vincentleung58** [comment]
I will go on a limb here and say you set catchup to False so the next job won't be triggered until 2024-02-01 assuming you tried 3 hours ago

**Reply by mike-soukup-vividseats** [reply]
@vincentleung58 Catchup is already set to False. Do you suggest setting to True? Maybe this can help validate if the scheduler will even create a monthly run timetable...

**Reply by vincentleung58** [reply]
> @vincentleung58 Catchup is already set to False. Do you suggest setting to True? Maybe this can help validate if the scheduler will even create a monthly run timetable...

Yes. Set it to True

**Reply by vincentleung58** [reply]
@mike-soukup-vividseats did it work?
